typhos/cli.py

```diff
@@ -6,60 +6,130 @@
 import pcdsutils
 from ophyd.sim import clear_fake_device, make_fake_device
+from pydm.widgets.template_repeater import FlowLayout
+from qtpy import QtCore, QtWidgets
 
 import typhos
 from typhos.app import get_qapp, launch_suite
 from typhos.benchmark.cases import run_benchmarks
 from typhos.benchmark.profile import profiler_context
+from typhos.display import DisplayTypes, ScrollOptions
+from typhos.suite import TyphosSuite
 from typhos.utils import nullcontext
 
 logger = logging.getLogger(__name__)
 
 # Argument Parser Setup
-parser = argparse.ArgumentParser(description='Create a TyphosSuite for '
-                                             'device/s stored in a Happi '
-                                             'Database')
-
-parser.add_argument('devices', nargs='*',
-                    help='Device names to load in the TyphosSuite or '
-                         'class name with parameters on the format: '
-                         'package.ClassName[{"param1":"val1",...}]')
-parser.add_argument('--happi-cfg',
-                    help='Location of happi configuration file '
-                         'if not specified by $HAPPI_CFG environment variable')
-parser.add_argument('--fake-device', action='store_true',
-                    help='Create fake devices with no EPICS connections. '
-                         'This does not yet work for happi devices. An '
-                         'example invocation: '
-                         'typhos --fake-device ophyd.EpicsMotor[]')
-parser.add_argument('--version', '-V', action='store_true',
-                    help='Current version and location '
-                         'of Typhos installation.')
-parser.add_argument('--verbose', '-v', action='store_true',
-                    help='Show the debug logging stream')
-parser.add_argument('--dark', action='store_true',
-                    help='Use the QDarkStyleSheet shipped with Typhos')
-parser.add_argument('--stylesheet',
-                    help='Additional stylesheet options')
-parser.add_argument('--profile-modules', nargs='*',
-                    help='Submodules to profile during the execution. '
-                         'If no specific modules are specified, '
-                         'profiles all submodules of typhos. '
-                         'Turns on line profiling.')
-parser.add_argument('--profile-output',
-                    help='Filename to output the profile results to. '
-                         'If omitted, prints results to stdout. '
-                         'Turns on line profiling.')
-parser.add_argument('--benchmark', nargs='*',
-                    help='Runs the specified benchmarking tests instead of '
-                         'launching a screen. '
-                         'If no specific tests are specified, '
-                         'runs all of them. '
-                         'Turns on line profiling.')
+parser = argparse.ArgumentParser(
+    description='Create a TyphosSuite for '
```

Comment:
This feels medium-high-level nitpicky, but I think there is room for improvement. Consider this instead:

```python
parser.add_argument(
    'devices',
    nargs='*',
    help=(
        "Device names to load in the TyphosSuite or class name with "
        "parameters of the format: package.ClassName[{"param1":"val1",...}]."
    ),
)
```

Primarily: `(` + newline + strings at same indent level, and final closing-line ")"

Reply:
You're definitely right here. I thought I was being clever by just running `black` and hoping for the best, but I didn't think much about the formatting afterwards.